src/app/course/search-course/search-course.component.spec.ts

```diff
@@ -1,25 +1,35 @@
-import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { async, ComponentFixture, TestBed } from "@angular/core/testing";
 
-import { SearchCourseComponent } from './search-course.component';
+import { SearchCourseComponent } from "./search-course.component";
+import { CUSTOM_ELEMENTS_SCHEMA, NO_ERRORS_SCHEMA } from "@angular/core";
+import { FormsModule, ReactiveFormsModule } from "@angular/forms";
 
-describe('SearchCourseComponent', () => {
-  let component: SearchCourseComponent;
-  let fixture: ComponentFixture<SearchCourseComponent>;
+describe("SearchCourseComponent", () => {
+    let component: SearchCourseComponent;
+    let fixture: ComponentFixture<SearchCourseComponent>;
 
-  beforeEach(async(() => {
-    TestBed.configureTestingModule({
-      declarations: [ SearchCourseComponent ]
-    })
-    .compileComponents();
-  }));
+    beforeEach(async(() => {
+        TestBed.configureTestingModule({
+            declarations: [SearchCourseComponent],
+            providers: [FormsModule, ReactiveFormsModule],
+            schemas: [CUSTOM_ELEMENTS_SCHEMA, NO_ERRORS_SCHEMA]
+        }).compileComponents();
+    }));
 
-  beforeEach(() => {
-    fixture = TestBed.createComponent(SearchCourseComponent);
-    component = fixture.componentInstance;
-    fixture.detectChanges();
-  });
+    beforeEach(() => {
+        fixture = TestBed.createComponent(SearchCourseComponent);
+        component = fixture.componentInstance;
+        fixture.detectChanges();
+    });
 
-  it('should create', () => {
-    expect(component).toBeTruthy();
-  });
+    it("should create", () => {
+        expect(component).toBeTruthy();
+    });
+
+    it("should emit an event upon searching for a course", () => {
+        spyOn(component.courseSearched, "emit");
```

Comment:
also, you can use
`const emitSpy: jasmine.Spy = spyOn(component.courseSearched, "emit");`
... your 2 lines, and then in expect:
`expect(emitSpy).toHaveBeenCalledWith("ABC");`